src/compiler/tsc.ts

```diff
@@ -102,6 +102,20 @@ namespace ts {
         }
     }
 
+    function reportWatchDiagnostic(diagnostic: Diagnostic) {
+		let output = new Date().toLocaleTimeString() + " - ";
+		
+        if (diagnostic.file) {
+            let loc = getLineAndCharacterOfPosition(diagnostic.file, diagnostic.start);
+

```

Comment:
My bad. PhpStorm is too slow with TypeScript project on my computer. I ended up using Notepad++ that's why the tab character slipped in. :)